Sources/AccessibilitySnapshot/Core/Swift/Classes/AccessibilitySnapshotView.swift

```diff
@@ -447,11 +483,18 @@ private extension AccessibilitySnapshotView {
                     locale: locale
                 )
             }
-
+            
+            static func moreContentAvailableText(for locale: String?) -> String {
+                return "More Content Available".localized(
+                    key: "custom_content.description",
+                    comment: "Description for an accessibility element indicating that it has additional custom content available",
+                    locale: locale
+                )
```

Comment:
This needs to be added to the localized strings files (why the Validate Localized Strings job is failing). There are schemes for the demo app to launch in German and Russian so you can pull the translations from what VoiceOver says.